uswds/uswds
# USWDS - Alert: Repair build:html errors related to comparison test files

## Summary
Repaired the `build:html` error that was introduced in PR https://github.com/uswds/uswds/pull/4922

## Related issue
Closes #4967

## Preview link
Preview link: [Alert comparison page](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.sites.pages.cloud.gov/preview/uswds/uswds/al-buildhtml-error/?path=/story/components-alert--alert-comparison)

## Problem statement
The file structure for test-alert-comparison is causing errors when running `npm run build:html`
```
❯ npm run build:html

> @uswds/uswds@3.1.0 build:html
> webpack build --config ./webpack.twig.config.js && npm run prettier:html

assets by status 656 KiB [cached] 183 assets
./html-templates/main.js 1 bytes [built] [code generated]

ERROR in Conflict: Multiple assets emit different content to the same filename test-alert-comparison.html

webpack 5.58.1 compiled with 1 error in 3667 ms
```

## Solution
Moving the contents from the JSON data file into the twig file prevents the duplicate file error. 

## Testing and review
To test:
1. checkout the `al-buildhtml-error`
1. Run `npm run build:html` and confirm no errors
1. Run `npm start` and confirm that the [Alert comparison page ](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.sites.pages.cloud.gov/preview/uswds/uswds/al-buildhtml-error/?path=/story/components-alert--alert-comparison)matches what is found in `develop`

---

Before opening this PR, make sure you’ve done whichever of these applies to you:
